app/service_routes.js

```diff
@@ -95,4 +95,21 @@ routes.options('/services/images', cors())
  */
 routes.get('/services/images', cors(), jwtCheck, resources.services.images.get)
 
+// Catch all for all broken api paths, direct to 404 response.
+routes.get('/services/*', (req, res) => {
+  res
+    .status(404)
+    .json({ status: 404, error: 'Not found. Did you mispell something?' })
+})
+routes.post('/services/*', (req, res) => {
+  res
+    .status(404)
+    .json({ status: 404, error: 'Not found. Did you mispell something?' })
+})
+routes.put('/services/*', (req, res) => {
+  res
+    .status(404)
+    .json({ status: 404, error: 'Not found. Did you mispell something?' })
+})
+
```

Comment:
See above comment under API routes, regarding using `app.all()` to handle all HTTP verbs. I don't mind having a separate handler for services and APIs.